hack/lib/golang.sh

```diff
@@ -240,6 +239,7 @@ kube::golang::set_platform_envs() {
     if [[ ${platform} == "linux/arm" ]]; then
       export CGO_ENABLED=1
       export CC=arm-linux-gnueabi-gcc
+      export GOROOT=${K8S_PATCHED_GOROOT}
```

Comment:
Yes, building **on** linux/arm with normally released Go is obviously broken (where the patched Golang doesn't exist)
Not much we can do about that for the meantime.